### PR TITLE
Link configuration for resource to container

### PR DESCRIPTION
**Pull request by c0ffeec0der**
Fix for issue #100

### CODE REVIEW

**Comment by c0ffeec0der** [comment]
@brancz Hello, could you help to merge this minor fix. Thank you

File: grafana/grafana.libsonnet [hunk 1]
```diff
@@ -295,10 +295,7 @@
         readinessProbe: {
           httpGet: { path: '/api/health', port: portName },
         },
-        resources: {
-          requests: { cpu: '100m', memory: '100Mi' },
-          limits: { cpu: '200m', memory: '200Mi' },
-        },
+        resources: $._config.grafana.container,
```
Comment (brancz) [inline grafana/grafana.libsonnet:298]: actually .. could you rename this field from `container` to `resources`? I'm not sure how that got in the configuration this way

Comment (c0ffeec0der) [reply grafana/grafana.libsonnet:298]: Yep sure, I was thinking the same thing but was worried others code might already use this part. Will do in a sec

**Comment by c0ffeec0der** [comment]
@brancz fix is done, please review. Thank you

**Review by brancz — APPROVED** [review]
Thanks!

**Pull request merged**